SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -252,7 +252,15 @@ FNetworkGUID FSpatialNetGUIDCache::AssignNewStablyNamedObjectNetGUID(UObject* Ob
 		UE_LOG(LogSpatialPackageMap, Fatal, TEXT("Found object called PersistentLevel which isn't a Level! This is not allowed when using the GDK"));
 	}
 
-	FUnrealObjectRef StablyNamedObjRef(0, 0, Object->GetFName().ToString(), (OuterGUID.IsValid() && !OuterGUID.IsDefault()) ? GetUnrealObjectRefFromNetGUID(OuterGUID) : FUnrealObjectRef());
+	bool bNoLoad = false;
+	if (IsNetGUIDAuthority())
```

Comment:
For readability purposes, could this condition be explicit in that it's trying to see if we are the server or not?

Reply:
I think using this function is fine.

Reply:
I guess Unreal uses this because it will be true when the driver is null (inside it's literally `Driver == NULL || Driver->IsServer()`)
I'm okay with either, but if we decide to use an explicit call to `IsServer` we should do that everywhere, so I'll leave this here as is for now.